### DISCUSSION TITLE
[Regression] Error when trying to access HDR environment/skybox with absolute path

### DISCUSSION THREAD

**Discussion by hybridherbst** [google/model-viewer#2219] [Q&A]
### Description

This was working fine in the past but seems to not work right now, with an error saying
`model-viewer.js:63249 Uncaught (in promise) Event` (seems in env generation)

![image](https://user-images.githubusercontent.com/2693840/92227753-02ac0b00-eea7-11ea-9eec-69c0eaeb1273.png)

and resulting in a black rendering (as if the image was not found).
![image](https://user-images.githubusercontent.com/2693840/92227930-574f8600-eea7-11ea-823c-593af84b9e66.png)

#### Live Demo
https://glitch.com/edit/#!/olivine-sturdy-hawk

### Version

- model-viewer: v1.2.1

### Browser Affected

- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [X] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [X] Windows

**Comment by elalish** [comment]
I've run across this before with Glitch. I have to remove the `?v=...` from the end of the URL to get it to work. I'm not quite sure why three chokes on the URL wit the query param, but it could be a bug on their side.

**Reply by SSGDoc** [reply]
![sss](https://user-images.githubusercontent.com/61503893/116408045-abdab380-a832-11eb-8c43-bbd862c3efb8.PNG)

The problem seems to be something related to using a full path vs. relative path. If i use relative path to the .hdr, it wont load the model at all, and give me a "Bad file format". Is there something we could do about this?

**Reply by elalish** [reply]
Can you post a link I can repro with? What you're describing should work.
